xmlWorksheet.go

```diff
@@ -313,20 +313,20 @@ type xlsxSheetData struct {
 // xlsxRow directly maps the row element. The element expresses information
 // about an entire row of a worksheet, and contains all cell definitions for a
 // particular row in the worksheet.
-type xlsxRow struct {
+type xlsxRow struct {	// alignment word
```

Comment:
struct fields order should be consistent with defined in XSD with ECMA-376 Spec: [`CT_Row`](https://github.com/xuri/xsd/blob/b7fd4e5c7e0d2ad785de32e140161ea5782f5d24/xsd/OfficeOpenXML-XMLSchema-Strict/sml.xsd#L2238-L2255).

Reply:
done